compiler/rustc_transmute/Cargo.toml

```diff
@@ -14,6 +14,7 @@ rustc_macros = { path = "../rustc_macros", optional = true}
 rustc_middle = { path = "../rustc_middle", optional = true}
 rustc_span = { path = "../rustc_span", optional = true}
 rustc_target = { path = "../rustc_target", optional = true}
+rustc_transmute_types = { path = "../rustc_transmute_types", optional = true}
```

Comment:
I don't think this can be optional. To check, `cd` into `rustc_transmute` and run `cargo check`.

Reply:
thanks. In my latest commit I fixed the build (had to gate some more code behind the rustc flag because we depend on rustc_middle more)